### ISSUE TITLE
The contents in the Own GeoKrety folders are mixed up with Moves of my GeoKrety in your personal account

### ISSUE THREAD

**Issue by Detroit70** [geokrety/geokrety-website#862]
The contents in the Own GeoKrety folders are mixed up with Moves of my GeoKrety in your personal account
Now if you are in your personal account and go to the Own GeoKrety folder, this folder displays geocretys, which in the old version you could see in the Moves of my GeoKrety folder
If you go from your account to the Moves of my GeoKrety folder, then after clicking on this folder you will be taken to the main page of the Geokrety website
Concluding the above, it turns out that there is completely no information on all the moles created by the player, which were previously located in the Own Geokrety folder
Geokrety located in the Own GeoKrety folder should be in the Moves of my GeoKrety folder
![wrong](https://github.com/geokrety/geokrety-website/assets/51854774/86554ddd-b924-495a-a78a-dfb72015d5d3)
In the Moves of my GeoKrety Folder, after going to this folder, information about the movement of geokrety should be displayed (now go to the main page of the site)
![empty](https://github.com/geokrety/geokrety-website/assets/51854774/8039607b-8864-4d4b-8162-5eb9fa11e395)

**Comment by kumy** [comment]
I'm sorry I don't get the issue here.

* If I look to my [Own GeoKrety](https://geokrety.org/en/users/26422/owned-geokrety):

This is the list of my GeoKrety sorted by last geokrety update. I see the last position of each of them
![image](https://github.com/geokrety/geokrety-website/assets/176794/c13a32a6-7159-46ea-b1a8-807c0af7ef40)


* If I look to my [Moves of my GeoKrety](https://geokrety.org/en/users/26422/owned/recent-moves)

This is the lists of all moves done on my GeoKrety
![image](https://github.com/geokrety/geokrety-website/assets/176794/c8de6713-05e0-41e5-9947-7f9963f72225)


It seems to be fine for my account. Could you also share the links of the pages from your screenshots

**Comment by Detroit70** [comment]
2023-10-06 23-22-05.MP4
<https://drive.google.com/file/d/1hSYgpT7cXp1-6CGvXYp1gmorQkHYwOlo/view?usp=drive_web>
I am attached video file, when  i click on buttom from my account - moves
on my geokrety -   I am redirected to the main page of the site
I am attached screen from old site from folder Own Geokrety, This
screen have a name - old version.
And last screen from new site from folder Own Geokrety. From my opinion I
can see in this folder now that when any player tags my geokrety it is
displayed in the folder
Link
https://geokrety.org/ru/users/23452/owned-geokrety?
As an a last example
[image: значок статуса] Архитектура Праги 1978 - мосты
<https://geokrety.org/ru/geokrety/GK1519E>
GK1519E  MS3977 <https://geokrety.org/go2geo/?wpt=MS3977>
<https://geokrety.org/ru/geokrety/GK1519E#log4798189> 4 часа назад
lisichka_cheff <https://geokrety.org/ru/users/47574> 14511 kmbut I can't
find the latest geokrety which i was created
Link on page Moves of my GeoKrety
<https://geokrety.org/ru/users/23452/owned/recent-moves> i didn not have
because i redirect on main page

пт, 6 окт. 2023 г. в 22:40, kumy ***@***.***>:

**Comment by Detroit70** [comment]
Yes, I understand, now I had to click on ID all is ok and i can saw all my
geokrety
Then everything will be like in the old folder.
Sorry for this moment!
But the folder Moves of my GeoKrety
<https://geokrety.org/ru/users/23452/owned/recent-moves> takes me to the
main page

сб, 7 окт. 2023 г. в 00:09, Федор Попов ***@***.***>:

**Comment by kumy** [comment]
I can reproduce the blank page error with your account (with English language selected). And the redirect of the same page to the home when in Russian... I will check the logs to get the underling error. I suspect a null value somewhere (which php 8.2 is not keen of compared to php7).

**Comment by kumy** [comment]
Underlying problem is a memory exhaustion :cry: 
```
Internal Server Error

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes)

[/var/www/geokrety/vendor/bcosca/fatfree-core/db/sql/mapper.php:661] 
```

**Comment by kumy** [comment]
This is a really annoying bug. I've tracked it a bit and it seems related to the ORM loading all the data at once. It will take time to fix :(
